test/integration/full/smoke.spec.js

```diff
@@ -0,0 +1,28 @@
+import { describe, it, expect } from '@jest/globals';
+import { configureServer } from '../../../src/server';
+import supertest from 'supertest';
+import * as cheerio from 'cheerio';
+
+describe('Smoke test', () => {
+  let server;
+  let request;
+
+  beforeEach(async () => {
+    server = await configureServer();
+    await server.start();
+    request = supertest(`http://localhost:${server.info.port}`);
```

Comment:
When these tests run in the Jenkins pipeline, is the test executed in the same container as the app?

Reply:
Yes the test is in the same container